### PR TITLE
[CDF-822] Review the CDF API javascript documentation for correctness

### PR DESCRIPTION
**Pull request by afrjorge**

### CODE REVIEW

**Comment by afrjorge** [comment]
@diogofscmariano please review

File: core-js/src/main/javascript/cdf/dashboard/Dashboard.i18n.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*!
- * Copyright 2002 - 2016 Webdetails, a Pentaho company. All rights reserved.
+ * Copyright 2002 - 2015 Webdetails, a Pentaho company. All rights reserved.
```
Comment (jvelasques) [inline core-js/src/main/javascript/cdf/dashboard/Dashboard.i18n.js]: @afrjorge was this intended?

File: core-js/src/main/javascript/cdf/dashboard/Dashboard.js [hunk 2]
```diff
@@ -199,8 +196,8 @@ define([
 
 
     /**
-     * @summary Determines if components and params are retrieved from the _window_ object or from the dashboard instance
-     * @description Determines if components and params are retrieved from the _window_ object or from the dashboard instance
+     * @summary Determines if components and parameters are retrieved from the _window_ object or from the dashboard instance.
```
Comment (jvelasques) [inline core-js/src/main/javascript/cdf/dashboard/Dashboard.js]: should we replace _ with ` here as well ?

**Comment by afrjorge** [comment]
@diogofscmariano @jvelasques updated and squashed

**Comment by afrjorge** [comment]
@diogofscmariano @jvelasques updated and squashed, please review

**Pull request merged**